compiler/src/codegen/compcore.re

```diff
@@ -3506,7 +3461,6 @@ let compile_wasm_module =
     ignore @@ compile_imports(wasm_mod, env, prog, import_map);
     ignore @@ compile_globals(wasm_mod, env, prog);
     ignore @@ compile_functions(wasm_mod, env, prog);
-    ignore @@ compile_exports(wasm_mod, env, prog);
```

Comment:
I feel like this should be a breaking change, given were changing the module signature.

Reply:
It's breaking in the sense that the module signature will be different than it currently is on the `main` branch, but the behavior on main is a regression and this just restores the signature to the way it worked before custom object files were merged.